extensions/wikia/PhalanxII/classes/PhalanxStatsWikiaPager.class.php

```diff
@@ -10,20 +12,44 @@ public function __construct( $id ) {
 	}
 
 	function formatRow( $row ) {
-		$type = implode( ", ", Phalanx::getTypeNames( ( isset( $row->ps_blocker_hit ) ) ? $row->ps_blocker_hit : $row->ps_blocker_type ) );
+		$blockId = (int) $row->ps_blocker_id;
+
+		// Get row data
+		$type = $this->buildTypeNames( $row );
 		$username = $row->ps_blocked_user;
+		$phalanxUrl = $this->buildPhalanxUrl( $blockId );
 		$timestamp = $this->app->wg->Lang->timeanddate( $row->ps_timestamp );
+		$statsUrl = $this->buildStatsUrl( $blockId );
 		$url = $row->ps_referrer;
-		$blockId = (int) $row->ps_blocker_id;
-		# block
-		$phalanxUrl = $this->mSkin->makeLinkObj( $this->mTitle, $blockId, 'id=' . $blockId );
-		# stats
-		$statsUrl = $this->mSkin->makeLinkObj( $this->mTitleStats, wfMsg( 'phalanx-link-stats' ), 'blockId=' . $blockId );
 
-		$html  = Html::openElement( 'li' );
-		$html .= wfMsgExt( 'phalanx-stats-row-per-wiki', array( 'parseinline', 'replaceafter' ), $type, $username, $phalanxUrl, $timestamp, $statsUrl, $url );
+		// Format the data with the phalanx-stats-row-per-wiki key and wrap it in an <li> tag
+		$html = Html::openElement( 'li' );
+		$html .= wfMsgExt(
+			'phalanx-stats-row-per-wiki',
+			[ 'parseinline', 'replaceafter' ],
+			$type, $username, $phalanxUrl, $timestamp, $statsUrl, $url
+		);
 		$html .= Html::closeElement( 'li' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$html". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)